src/ramsesio.cxx

```diff
@@ -288,6 +296,9 @@ Int_t RAMSES_get_nbodies(char *fname, int ptype, Options &opt)
     getline(Finfo,stringbuf);//time
     getline(Finfo,stringbuf);//a
     getline(Finfo,stringbuf);//hubble
+
+    getline(Finfo,stringbuf);//Mysterious mis-reading one line
+
```

Comment:
@JinsuRhee, Is this true for all RAMSES formats? Might it be necessary to wrap this in ifdefs and add some compilation options to the ramses format? This might apply to the additions you have added below.